server/src/services/comment.service.ts

```diff
@@ -124,6 +171,16 @@ export class CommentService {
     return foundComment;
   }
 
+  public async updateCommentUser(commentUserId: string, body: any) {
+    const item = await this.commentUsersRepository.update(commentUserId, { ...body });
+    return item;
+  }
+
+  public async updateAllCommentUser(body: any) {
+    const item = await this.commentUsersRepository.update({}, { ...body });
```

Comment:
please pick and pass the updatable params, passing entire body will cause security issues

Reply:
got it, I've updated